core/docker-entrypoint.sh

```diff
@@ -137,17 +128,47 @@ admin_pass: $MAILMAN_REST_PASSWORD
 class: mailman_hyperkitty.Archiver
 enable: yes
 configuration: /etc/mailman-hyperkitty.cfg
+
+EOF
+
+# Generate a basic configuration to use exim
+cat > /etc/exim-mailman.cfg <<EOF
+[mta]
+incoming: mailman.mta.exim4.LMTP
+outgoing: mailman.mta.deliver.deliver
+lmtp_host: $MM_HOSTNAME
+lmtp_port: 8024
+smtp_host: $SMTP_HOST
+smtp_port: $SMTP_PORT
+configuration: python:mailman.config.exim4
+
 EOF
 
 # Generate a basic configuration to use postfix.
 cat > /etc/postfix-mailman.cfg <<EOF
-[postfix]
```

Comment:
This is a different configuration that the one you replaced it with. This file is pointed to by `configuration` under `[mta]` section and holds some special config about a specific MTA, in this case, postfix.

You can keep this file around and have a different name for the temporary file that you create with some changes, see below.